src/data/defaults.txt

```diff
@@ -170,6 +170,7 @@ global	_gitUpdated	false
 global	_svnRepoFileFetched	false
 global	_svnUpdated	false
 
+user	8BitBonusTurns	5
```

Comment:
I like descriptive preference names.  I'd prefer a name like 8BitTurnsBeforeBonusChange even though I will get annoyed every time I have to type it.  We might want to revisit the name once we know exactly what we are keeping track of.  I wondering whether 8BitTurnsSpent is a better name if the user has to handle the mod 5 calculation.  Not a blocker though.